### PR TITLE
Fix: #177

### PR DESCRIPTION
**Pull request by dbanksdesign**
*Issue #, if available:* #177

*Description of changes:* Fixing the mobile nav bugs in our doc site I caused by messing with the CSS in docsify. The navigation in mobile should work now!

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/assets/styles.css [hunk 1]
```diff
@@ -386,6 +385,20 @@ section.cover .cover-main {
     max-width: 100px;
     max-height: 100px;
   }
+
+  body .sidebar-toggle { background-color: #fff; }
+  body.close .sidebar-toggle {
+    background-color: transparent;
+    padding: 1rem;
```
Comment (didoo) [inline docs/assets/styles.css]: are you sure you need this line? I see the padding declared also in `body .sidebar-toggle {}` and nothing overrides it (or maybe I am missing something).

File: docs/assets/styles.css [hunk 2]
```diff
@@ -386,6 +385,20 @@ section.cover .cover-main {
     max-width: 100px;
     max-height: 100px;
   }
+
+  body .sidebar-toggle { background-color: #fff; }
+  body.close .sidebar-toggle {
+    background-color: transparent;
```
Comment (didoo) [inline docs/assets/styles.css]: maybe you can move this declaration directly under `body .sidebar-toggle {}`? (currently has background transparent and you are overriding it.

File: docs/assets/styles.css (hunk 1 above)
Comment (dbanksdesign) [reply docs/assets/styles.css]: I need it because these styles are overriding another 3rd party CSS file :/

File: docs/assets/styles.css [hunk 3]
```diff
@@ -53,11 +53,11 @@ aside.sidebar {
   border: none;
   background-color: var(--theme-color-secondary);
   color: #fff;
-  width: 20%;
+  /* width: 20%; */
 }
 
 body.close .sidebar {
-  transform: translateX(-100%);
+  /* transform: translateX(-100%); */
```
Comment (dbanksdesign) [inline docs/assets/styles.css]: I need to just remove these comments

File: docs/assets/styles.css (hunk 2 above)
Comment (dbanksdesign) [reply docs/assets/styles.css]: Same, it's because I'm overriding a 3rd party CSS file

**Comment by chazzmoney** [comment]
Ah! LGTM!

**Pull request merged**